docker/Dockerfile

```diff
@@ -1,41 +1,39 @@
-# We use multi-stage here to unzip in an initial layer so we don't have to COPY and then RUN unzip (two layers). ADD can lead to larger layers as well.
-FROM busybox:1.35.0@sha256:02289a9972c5024cd2f083221f6903786e7f4cb4a9a9696f665d20dd6892e5d6 AS unpack
+# Additional build image to unpack the zip file and change the permissions without retaining large layers just for those operations
```

Comment:
Retaining large layers means bloating the image? 

I like the improved doc. Just a suggestion: Wouldn't it be a bit more explanatory if there is sth like:
```
# Image 1: Additional build image to

# Image 2: Actual ...
```
Just to kind of separate the logic of the file into two sections

Reply:
done